tests/wpt/mozilla/tests/mozilla/document_elementsFromPoint_ref.html

```diff
@@ -0,0 +1,3 @@
+<!doctype html>
```

Comment:
Also, just for reference, there's a `blank.html` file used to avoid having a lot of files like this, though this is no longer relevant with my previous comment.